django_celery_results/backends/database.py

```diff
@@ -7,13 +7,17 @@
 from celery.result import GroupResult, allow_join_result, result_from_tuple
 from celery.utils.log import get_logger
 from celery.utils.serialization import b64decode, b64encode
-from django.db import transaction
+from django.db import connection, transaction
+from django.db.utils import InterfaceError
 from kombu.exceptions import DecodeError
+from psycopg2 import InterfaceError as Psycopg2InterfaceError
```

Comment:
This adds a hard dependency on psycopg2, but django-celery-results can be used with other databases as well.

I have opened PR #292 to fix this.